google/cloud/aiplatform/datasets/__init__.py

```diff
@@ -16,6 +16,7 @@
 #
```

Comment:
This doesn't seem to refactor the TrainingJobs as much as it is a refactor of Tabular/Timeseries datasets. Was the intention also to refactor Tabular/Forecasting training jobs similarly?

Reply:
It moves most of the column names logic to a common place. I agree that it some should live in TrainingJob instead of Dataset. Will refactor.

Reply:
I moved the transformations-specific code to a util file instead of creating a superclass for TabularTrainingJob and ForecastingTrainingJob to inherit. It seemed simpler since there's no inherited state that they need to share.